packages/fscomponents/src/components/CartItem/VariantCartItem.tsx

```diff
@@ -0,0 +1,299 @@
+    alignItems: 'center',
+    marginTop: 5
+  },
+  outOfStockText: {
+    fontWeight: '500',
+    fontSize: 14,
+    lineHeight: 20,
+    letterSpacing: 0.5,
+    color: palette.error,
+    marginLeft: 5
+  },
+  priceWrapper: {
+    marginBottom: 7
+  },
+  detailLine: {
+    lineHeight: 20,
+    marginBottom: 2
+  }
+});
+
+const icons = {
+  error: require('../../../assets/images/error.png')
+};
+
+export interface VariantCartItemProps extends CommerceTypes.CartItem {
+  /**
+   * A function to invoke when the user wants to modify the quantity of an item in cart.
+   */
+  onQtyChange?: (itemId: string, qty: number) => Promise<void>;
+
+  /**
+   * A function to invoke when the user wants move the item to their wishlist.
+   *
+   * The move to wishlist button will only display if this is provided.
+   */
+  onMoveToWishlist?: (item: CommerceTypes.CartItem) => void;
+
+  /**
+   * A callback to invoke if the user taps on the image
+   */
+  onImagePress?: (item: CommerceTypes.CartItem) => void;
+
+  /**
+   * Additional props that can be applied to the 'move to wishlist' button
+   */
+  wishlistButtonProps?: Partial<ButtonProps>;
+
+  /**
+   * Styles to apply to the main container
+   */
+  style?: StyleProp<ViewStyle>;
+
+  /**
+   * Styles to apply to the product item title
+   */
+  titleStyle?: StyleProp<TextStyle>;
+
+  /**
+   * Styles to apply to the left column container
+   * (eg. the column with the product image and move to wishlist button)
+   */
+  leftColumnStyle?: StyleProp<ViewStyle>;
+
+  /**
+   * Styles to apply to the right column container
+   * (eg. the column with the product details and qty stepper)
+   */
+  rightColumnStyle?: StyleProp<ViewStyle>;
+
+  productImageStyle?: StyleProp<ImageStyle>;
+  originalPriceStyle?: StyleProp<TextStyle>;
+  priceStyle?: StyleProp<TextStyle>;
+
+  /**
+   * Text styles that will apply to the price if the item's price is less than originalPrice
+   */
+  salePriceStyle?: StyleProp<TextStyle>;
+
+  /**
+   * Text styles that will apply to the item variant detail label (eg. 'Size')
+   */
+  itemDetailLabelStyle?: StyleProp<TextStyle>;
+
+  /**
+   * Text styles that will apply to the item variant detail value (eg. 'Small')
+   */
+  itemDetailValueStyle?: StyleProp<TextStyle>;
+
+  outOfStockTextStyle?: StyleProp<TextStyle>;
+  stepperStyle?: StyleProp<ViewStyle>;
+
+  /**
+   * Additional props that will be provided to the stepper
+   */
+  stepperProps?: Partial<StepperProps>;
+}
+
+export interface SerializableVariantCartItemProps extends
+  Omit<VariantCartItemProps, keyof CommerceTypes.CartItem> {
+  style?: ViewStyle;
+  titleStyle?: TextStyle;
+  leftColumnStyle?: ViewStyle;
+  rightColumnStyle?: ViewStyle;
+  productImageStyle?: ImageStyle;
+  originalPriceStyle?: TextStyle;
+  priceStyle?: TextStyle;
+  salePriceStyle?: TextStyle;
+  itemDetailLabelStyle?: TextStyle;
+  itemDetailValueStyle?: TextStyle;
+  outOfStockTextStyle?: TextStyle;
+  stepperStyle?: ViewStyle;
+}
```

Comment:
These serializable props are missing a lot of required props. This seems to work:

```
interface BaseVariantCartItemProp extends CommerceTypes.CartItem {
  /**
   * Additional props that can be applied to the 'move to wishlist' button
   */
  wishlistButtonProps?: Partial<ButtonProps>;

  /**
   * Additional props that will be provided to the stepper
   */
  stepperProps?: Partial<StepperProps>;
  /**
   * Styles to apply to the main container
   */
  style?: StyleProp<ViewStyle>;

  /**
   * Styles to apply to the product item title
   */
  titleStyle?: StyleProp<TextStyle>;

  /**
   * Styles to apply to the left column container
   * (eg. the column with the product image and move to wishlist button)
   */
  leftColumnStyle?: StyleProp<ViewStyle>;

  /**
   * Styles to apply to the right column container
   * (eg. the column with the product details and qty stepper)
   */
  rightColumnStyle?: StyleProp<ViewStyle>;

  productImageStyle?: StyleProp<ImageStyle>;
  originalPriceStyle?: StyleProp<TextStyle>;
  priceStyle?: StyleProp<TextStyle>;

  /**
   * Text styles that will apply to the price if the item's price is less than originalPrice
   */
  salePriceStyle?: StyleProp<TextStyle>;

  /**
   * Text styles that will apply to the item variant detail label (eg. 'Size')
   */
  itemDetailLabelStyle?: StyleProp<TextStyle>;

  /**
   * Text styles that will apply to the item variant detail value (eg. 'Small')
   */
  itemDetailValueStyle?: StyleProp<TextStyle>;

  outOfStockTextStyle?: StyleProp<TextStyle>;
  stepperStyle?: StyleProp<ViewStyle>;
}

export interface VariantCartItemProps extends BaseVariantCartItemProp {
  /**
   * A function to invoke when the user wants to modify the quantity of an item in cart.
   */
  onQtyChange?: (itemId: string, qty: number) => Promise<void>;

  /**
   * A function to invoke when the user wants move the item to their wishlist.
   *
   * The move to wishlist button will only display if this is provided.
   */
  onMoveToWishlist?: (item: CommerceTypes.CartItem) => void;

  /**
   * A callback to invoke if the user taps on the image
   */
  onImagePress?: (item: CommerceTypes.CartItem) => void;
}

export interface SerializableVariantCartItemProps extends BaseVariantCartItemProp {
  style?: ViewStyle;
  titleStyle?: TextStyle;
  leftColumnStyle?: ViewStyle;
  rightColumnStyle?: ViewStyle;
  productImageStyle?: ImageStyle;
  originalPriceStyle?: TextStyle;
  priceStyle?: TextStyle;
  salePriceStyle?: TextStyle;
  itemDetailLabelStyle?: TextStyle;
  itemDetailValueStyle?: TextStyle;
  outOfStockTextStyle?: TextStyle;
  stepperStyle?: ViewStyle;
}
```

I went with that version because of how many styles would need to be omitted for the different styles.